public/js/icinga/behavior/navigation.js

```diff
@@ -319,7 +319,8 @@
             $target.siblings().not($target).removeClass('hover');
             $target.addClass('hover');
 
-            var targetHeight = $target.offset().top + $target.outerHeight();
+            let target = $target[0];
```

Comment:
placement is still off, especially once the menu is scrollable. Retrieve top offset and height by using getBoundingClientRect instead.